File: java-client/src/main/java/org/rakam/client/model/SearchUsers.java
package org.rakam.client.model;

import org.rakam.StringUtil;
import java.util.*;
import org.rakam.client.model.EventFilter;
import org.rakam.client.model.Sorting;



import io.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
@javax.annotation.Generated(value = "class io.swagger.codegen.languages.JavaClientCodegen", date = "2015-12-06T22:11:51.057+02:00")
public class SearchUsers   {
  
  private String project = null;
  private List<String> columns = new ArrayList<String>();
  private String filter = null;
  private List<EventFilter> eventFilters = new ArrayList<EventFilter>();
  private Sorting sorting = null;
  private Integer offset = null;
  private Integer limit = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("project")
  public String getProject() {
    return project;
  }
  public void setProject(String project) {
    this.project = project;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("columns")
  public List<String> getColumns() {
    return columns;
  }
  public void setColumns(List<String> columns) {
    this.columns = columns;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("filter")
  public String getFilter() {
    return filter;
  }
  public void setFilter(String filter) {
    this.filter = filter;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("event_filters")
  public List<EventFilter> getEventFilters() {
    return eventFilters;
  }
  public void setEventFilters(List<EventFilter> eventFilters) {
    this.eventFilters = eventFilters;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("sorting")
  public Sorting getSorting() {
    return sorting;
  }
  public void setSorting(Sorting sorting) {
    this.sorting = sorting;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("offset")
  public Integer getOffset() {
    return offset;
  }
  public void setOffset(Integer offset) {
    this.offset = offset;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("limit")
  public Integer getLimit() {
    return limit;
  }
  public void setLimit(Integer limit) {
    this.limit = limit;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class SearchUsers {\n");
    
    sb.append("    project: ").append(StringUtil.toIndentedString(project)).append("\n");
    sb.append("    columns: ").append(StringUtil.toIndentedString(columns)).append("\n");
    sb.append("    filter: ").append(StringUtil.toIndentedString(filter)).append("\n");
    sb.append("    eventFilters: ").append(StringUtil.toIndentedString(eventFilters)).append("\n");
    sb.append("    sorting: ").append(StringUtil.toIndentedString(sorting)).append("\n");
    sb.append("    offset: ").append(StringUtil.toIndentedString(offset)).append("\n");
    sb.append("    limit: ").append(StringUtil.toIndentedString(limit)).append("\n");
    sb.append("}");
    return sb.toString();
  }
}

File: csharp-client/src/main/csharp/Rakam/Client/Model/SearchUsers.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace Rakam.Client.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class SearchUsers {
    
    /// <summary>
    /// Gets or Sets Project
    /// </summary>
    [DataMember(Name="project", EmitDefaultValue=false)]
    public string Project { get; set; }

    
    /// <summary>
    /// Gets or Sets Columns
    /// </summary>
    [DataMember(Name="columns", EmitDefaultValue=false)]
    public List<string> Columns { get; set; }

    
    /// <summary>
    /// Gets or Sets Filter
    /// </summary>
    [DataMember(Name="filter", EmitDefaultValue=false)]
    public string Filter { get; set; }

    
    /// <summary>
    /// Gets or Sets EventFilters
    /// </summary>
    [DataMember(Name="event_filters", EmitDefaultValue=false)]
    public List<EventFilter> EventFilters { get; set; }

    
    /// <summary>
    /// Gets or Sets Sorting
    /// </summary>
    [DataMember(Name="sorting", EmitDefaultValue=false)]
    public Sorting Sorting { get; set; }

    
    /// <summary>
    /// Gets or Sets Offset
    /// </summary>
    [DataMember(Name="offset", EmitDefaultValue=false)]
    public int? Offset { get; set; }

    
    /// <summary>
    /// Gets or Sets Limit
    /// </summary>
    [DataMember(Name="limit", EmitDefaultValue=false)]
    public int? Limit { get; set; }

    

    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class SearchUsers {\n");
      
      sb.Append("  Project: ").Append(Project).Append("\n");
      
      sb.Append("  Columns: ").Append(Columns).Append("\n");
      
      sb.Append("  Filter: ").Append(Filter).Append("\n");
      
      sb.Append("  EventFilters: ").Append(EventFilters).Append("\n");
      
      sb.Append("  Sorting: ").Append(Sorting).Append("\n");
      
      sb.Append("  Offset: ").Append(Offset).Append("\n");
      
      sb.Append("  Limit: ").Append(Limit).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
